lib/msf/core/payload/python/meterpreter_loader.rb

```diff
@@ -264,28 +262,46 @@ def _s2b(text):
 		return[ord(c)for c in text]
 	def _b2s(binary):
 		return bytes(binary)
-	def _concat_list(a,b):
-		return a+bytes(b)
 else:
 	bytes=lambda s,e:s
+def _gmul(a, b):
+	r=0
+	while b:
+		if b&1: r^=a
+		a<<=1
+		if a>255: a^=0x11B
+		b>>=1
+	return r
+def _mix(n, vec):
+	return sum([_gmul(n,v)<<(24-8*shift) for shift,v in enumerate(vec)])
+def _ror32(n):
+	return (n&255)<<24|n>>8
+def _rcon():
+	return [_gmul(1, 1<<n) for n in xrange(30)]
```

Comment:
Use `range` instead of `xrange`, it's both more portable and one char smaller.

Reply:
Done and done.